api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/CardClient.java

```diff
@@ -0,0 +1,51 @@
+package uk.gov.dhsc.htbhf.claimant.service;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestClientException;
+import org.springframework.web.client.RestTemplate;
+import uk.gov.dhsc.htbhf.claimant.exception.CardClientException;
+import uk.gov.dhsc.htbhf.claimant.model.card.CardRequest;
+import uk.gov.dhsc.htbhf.claimant.model.card.CardResponse;
+
+/**
+ * Service for interacting with the card services api..
```

Comment:
Minor - extra full stop, I'm surprised that Checkstyle doesn't moan at this :-).